### PR TITLE
All buttons consistent

### PR DESCRIPTION
**Pull request by ehab-hassan**
### Description

Making all buttons consistent in design and color

### Changes

To make all buttons consistent
- we ll use primary(filled) and secodery(outline) colors for main action 
image

- ll use error(red) color filled or outlined for alert , cancel, confirm delete delete (any action need to alert the user)
image 
image

- ll use white for normal action like Close, Clear, Export
image

- ll use outlined rather than tonal for disabled buttons because tonal acting wired at light mode

### Related Issues

#1643

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Comment by maayarosama** [comment]
- I suggest using the white outline color for the close button in deleting dialogues to make it consistent with all the other close buttons,  or we can just make it a cancel instead of close button 
Screenshot from 2023-12-24 16-36-18

- The blue color of learn how button looks out of place, I suggest changing the color to make it go with our color pallette 
Screenshot from 2023-12-24 16-39-22

- The gray color of th
Screenshot from 2023-12-24 16-40-07
e no button in the Dao makes it look disabled, how about changing it to another color?

**Comment by ehab-hassan** [comment]
same white we used at close color
image

image
image

**Review by mohamedamer453 — COMMENTED** [review]
- The buttons in the delete confirmation dialog weren't updated.

   image

- The `Load Nodes` button wasn't updated.

   image

- The `+` icon in the adding a disk/adding a worker button should also match the other buttons. 

   image

- The `Generate SSH Keys` button wasn't updated.

   image

- In some solutions the `Deploy` button is in the primary colors while in other solutions it's in the normal action(white) color.

   image

   image

- Buttons in the `Remove Public Config` dialog weren't updated.

   image

**Comment by ehab-hassan** [comment]
all done
image
image
image
image

**Comment by ehab-hassan** [comment]
> * Deploy button in `Nextcloud` solution wasn't updated.
>   image

good catch :)
fixed
image

File: packages/playground/src/components/deployment_data_dialog.vue [hunk 1]
```diff
@@ -104,10 +104,10 @@
           </pre>
           </template>
         </v-card-text>
-        <v-card-actions>
+        <v-card-actions class="my-1">
           <v-spacer />
-          <v-btn color="error" variant="tonal" @click="$emit('close')">Close</v-btn>
-          <v-btn color="secondary" variant="tonal" @click="copy">Copy</v-btn>
+          <v-btn color="anchore" variant="outlined" @click="$emit('close')">Close</v-btn>
```
Comment (mohamedamer453) [inline packages/playground/src/components/deployment_data_dialog.vue:109]: Typo in anchor

File: packages/playground/src/dashboard/components/public_ips_table.vue [hunk 2]
```diff
@@ -40,9 +40,9 @@
         <v-toolbar color="primary" class="custom-toolbar">
           <p class="mb-5">Delete IP</p>
         </v-toolbar>
-        <v-card-text> Are you sure you want to delete IP {{ itemToDelete?.ip }}? </v-card-text>
+        <v-card-text class="text-subtitle-1"> You sure you want to delete IP {{ itemToDelete?.ip }}? </v-card-text>
```
Comment (mohamedamer453) [inline packages/playground/src/dashboard/components/public_ips_table.vue:43]: The question shouldn't start with `you`, imo the `are` should be re-added or it should be rephrased to `Do you want to delete this IP?`

File: packages/playground/src/dashboard/components/public_config.vue [hunk 3]
```diff
@@ -139,20 +139,20 @@
           <v-toolbar color="primary" dark class="custom-toolbar">
             <p class="mb-5">Remove Public Config</p>
           </v-toolbar>
-          <v-card-text> Are you certain you want to remove this node's public config? </v-card-text>
+          <v-card-text>Remove this node's public config? </v-card-text>
```
Comment (mohamedamer453) [inline packages/playground/src/dashboard/components/public_config.vue:142]: same as the comment below, it shouldn't start with remove, imo it should be the same as before or rephrased to `Do you want to remove this node's public config?`

File: packages/playground/src/dashboard/components/public_ips_table.vue (hunk 2 above)
Comment (ehab-hassan) [reply packages/playground/src/dashboard/components/public_ips_table.vue]: image
fixed

also the typo

**Review by mohamedamer453 — APPROVED** [review]

**Review by AlaaElattar — APPROVED** [review]

**Pull request merged**